buddy_init/base_skeleton/requirements.txt

```diff
@@ -0,0 +1,3 @@
+jax
+jaxlib
+git+https://github.com/ministry-of-silly-code/experiment_buddy.git@main#egg=experiment_buddy
```

Comment:
I had a problem with requests. maybe check that too?

Reply:
I didn't get what you said

Reply:
if you install from scratch everything, the package requests was giving me trouble. can you check?